demo/react/doc/product/components/autocomplete.mdx

```diff
@@ -0,0 +1,114 @@
+```javascript import
+import { Autocomplete, List, ListItem, ListItemSize, Size, TextField, Chip, ListDivider, Icon, ListSubheader } from 'LumX';
+import { useBooleanState } from 'LumX/core/react/hooks';
+import { DOWN_KEY_CODE, ENTER_KEY_CODE, TAB_KEY_CODE, UP_KEY_CODE } from 'LumX/core/constants';
```

Comment:
There is many unused imports here

Reply:
yeah, I removed them on the next component: https://github.com/lumapps/design-system/pull/194/commits/983a0a7616cfa14e2de9caf1ec8150028a8177d6